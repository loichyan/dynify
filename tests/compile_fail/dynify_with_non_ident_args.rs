#[dynify::dynify]
trait Trait {
    async fn test(&self, (arg): String, NewType(inner): NewType);
}

fn main() {}
