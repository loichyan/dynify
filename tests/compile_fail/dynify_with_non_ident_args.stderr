error: typed argument must be a valid identifier
 --> tests/compile_fail/dynify_with_non_ident_args.rs:3:26
  |
3 |     async fn test(&self, (arg): String, NewType(inner): NewType);
  |                          ^^^^^
